pkg/cmd/infra/deployer/deployer.go

```diff
@@ -182,7 +183,8 @@ func (d *Deployer) Deploy(namespace, rcName string) error {
 	}
 
 	// Decode the config from the deployment.
-	config, err := appsutil.DecodeDeploymentConfig(to)
+	// TODO: Remove this once we are sure there are no internal versions of configs serialized in DC
+	config, err := appsinternalutil.DecodeDeploymentConfig(to)
```

Comment:
@deads2k this is what I was talking about in the other comment... if we decode to external version and then just return apps/v1/DC here we can get rid of appsinternalutil import and thus make the resulting binary smaller.